dev/reference/tools/data-migration/from-aurora.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Data Migration from AWS Aurora MySQL to TiDB
+summary: Using DM to migrate from AWS Aurora MySQL to TiDB
+category: tools
+---
+
+# Data Migration from AWS Aurora MySQL to TiDB
+
+This document describes how to use DM to migrate from [AWS Aurora MySQL](https://aws.amazon.com/cn/rds/aurora/details/mysql-details/) to TiDB。
+
+## Step 1: Start binlog in Aurora cluster
+
+This document assumes you'll migrate data from two Aurora clusters to  TiDB. The information list of Aurora clusters is as belowed. It indicates Aurora-1 has a single built-in reader endpoint. 
+
+| Cluster | Endpoint | Port | Role |
+|:-------- |:--- | :--- | :--- |
+| Aurora-1 | pingcap-1.h8emfqdptyc4.us-east-2.rds.amazonaws.com | 3306 | Writer |
+| Aurora-1 | pingcap-1-us-east-2a.h8emfqdptyc4.us-east-2.rds.amazonaws.com | 3306 | Reader |
+| Aurora-2 | pingcap-2.h8emfqdptyc4.us-east-2.rds.amazonaws.com | 3306 | Writer |
+
+During the incremental backup process, you need to set the binglog format as `ROW`. Inactive binlog or incorrect binlog format will hinder data migration using DM. See [Checking items](https://www.pingcap.com/docs-cn/tools/dm/precheck/#%E6%A3%80%E6%9F%A5%E5%86%85%E5%AE%B9) for more information.
+
+> **Warning:** 
+>
+> Aurora reader cannot start binlog, so it cannot be the upstream master server when using DM to migrate data.
```

Comment:
```suggestion
> Aurora reader cannot start binlog, so it cannot be the upstream master server when using DM to migrate data.
```
```suggestion
> Because binlog cannot be enabled in the Aurora reader, it cannot be taken as the upstream master server when you use DM to migrate data.
```